packages/runtime-dom/src/helpers/useCssVars.ts

```diff
@@ -47,6 +48,10 @@ export function useCssVars(getter: (ctx: any) => Record<string, string>) {
     watchPostEffect(setVars)
   })
 
+  onBeforeUpdate(() => {
+    watchPostEffect(setVars)
```

Comment:
Should just call `setVars` and not create another watcher on each update.

Reply:
When I just call setVars before update, the vnode is still a v-if comment, and it doesn't make sense to set custom properties for this comment.

![image](https://github.com/user-attachments/assets/afdb7ad2-4db5-497c-8cec-0069da072bee)

 But when I combine it with watchPostEffect, the vnode is correct.
 
![image](https://github.com/user-attachments/assets/eabdc8da-222e-430e-b828-f4ff9c5fbdd4)

Reply:
or use `nextTick` instead ?

Reply:
or use `queuePostFlushCb` instead?

Reply:
thanks